localization/english/culture/wc_basic_innovations_l_english.yml

```
@@ -0,0 +1,27 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ innovation_animal_war_training:0 "Animal War Training"
+ innovation_animal_war_training_desc:0 "Through skills taught to us my the best beastmasters, we can communicate, train and discipline the animals around us."
+ 
+ innovation_rogues_guild:0 "Rogues Guild"
+ innovation_rogues_guild_desc:0 "."
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+ innovation_rogues_school:0 "School of the Rogue"
+ innovation_rogues_school_desc:0 "."
```

Another proposal for loc.
```suggestion
 innovation_rogues_school_desc:0 "To be trained as a rogue is a difficult, frequently deadly quest. Students are taught the use of tools, combat techniques, evasion, and perhaps most infamously, assassination. Schools specializing in the art of roguery are rare, but almost uniformly produce skilled agents, should one overlook the failed students."
```